data/dialog.yaml

```
@@ -153,28 +153,32 @@ Gerud Ancient Dungeon Reward:
+      - print(While you walk into the Los Gerudos grand avenue, and admire the sea, visible from a long distance, you hear a noise, coming from behind you. You slowly turn behind. You find a dog; he must have followed you since you've left that little jewelry store. He must have been lost by his dear owners!)
       - print(You decide to take a look...)
       - wait(2)
       - "print(You found a golden collar on his neck, with inscribed letters on it. It says: “Mitchell - property of the Coopers; Please return this dog if you find him at the 13 Yasward Drive, Los Gerudos“. The dog looks at you with big eyes, seeming like he wants you to take him back to its owners.\n)"
-  use actions: false
+  use actions: true
+  actions:
+    add to diary:
+      known npcs:
+      - Marry Joselina
 
 Given Back Dog Mitchell:
   conversation:
     - label 1:
       - print(You finally arrive to the 13 Yasward Drive, Los Gerudos, the village of the Coopers. You knock at the door.)
       - wait(2)
-      - print(A bulter finally opens the great doors, and greets you, before asking who you are and what do you want. You explain to him everything, before showing him the dog. Suddently, the bulter shouts:)
-      - print(“Mrs. Cooper! We've found Mitchell back!“\n)
+      - print(A bulter finally opens the great doors, and greets you, before asking who you are and what do you want. You explain to him everything, before showing him the dog. Suddenly, the bulter shouts:)
+      - print(“- Mrs. Cooper! We've found Mitchell back!“\n)
       - wait(1)
-      - print(A chubby lady suddently goes down the main stairs, running towards the opened doors.)
-      - print(“Mitchell! Oh my godness! Finally! you're here! Where did you find him bulter?“, says the chubby lady.)
-      - print(“Actually it's me who've found him, in the grand avenue. I read his collar and then I came here.“, you respond to her.)
-      - print(“What is your name? I'm so grateful, and I couldn't be more.“)
-      - print(“My name is $name, and I'm from the Ayar Town, in the far Forlindon Woods. But I'm pretty much sure you haven't heard of this place before.“)
-      - print(“The Forlindon Woods? That is where I used to live, when I was a little girl! But then my dad bought a ranch there. What can I do for you? Is there anything you'd like to have?“, the lady asks)
-      - print(“Please, I dind't do much.“)
-      - print(“You didn't do much? You've saved my dear love! Please, accept these two ingots.“)
+      - print(A chubby lady suddenly goes down the main stairs, running towards the opened doors.)
+      - print(“- Mitchell! Oh my godness! Finally! you're here! Where did you find him bulter?, says the chubby lady.)
+      - print(- Actually it's me who've found him, in the grand avenue. I read his collar and then I came here., you respond to her.)
```

```suggestion
      - print("- Actually it's me who've found him, in the grand avenue. I read his collar and then I came here," you respond to her.)
```
How do you want to use quotes here?

---

The quotes start at the beginning of the conversation and finish at the end, so the quotes are only located at the first `print()` method of the conversation and the final one. I'm talking about the characters conversation here.